criu-3.15/lib/py/cli.py

```diff
@@ -384,7 +386,7 @@ def dump_stackmap_data(opts):
     section = elf_utils.get_elf_section(
         elffile, stack_map_utils.STACKMAP_SECTION)
     print("Reading section: " + section.name)
-    stack_maps = stack_map_utils.parse_stack_maps(section)
+    stack_maps, _ = stack_map_utils.parse_stack_maps(section)
```

Comment:
Why the anonymous var (_)? The function returns a single output not a tuple

Reply:
The function was modified to return stack map information to modify the records after shuffle. Process migration through `crit recode` has been tested after this change.